src/server/pfs/server/driver.go

```diff
@@ -460,7 +460,7 @@ func (d *driver) deleteRepoInfo(ctx context.Context, txnCtx *txncontext.Transact
 	// against certain corruption situations where the RepoInfo doesn't
 	// exist in postgres but branches do.
```

Comment:
Note to self, in the new relational world, branches cannot exist without the repos.